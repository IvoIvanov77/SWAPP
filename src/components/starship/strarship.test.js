import React from 'react';
import { shallow } from 'enzyme';
import Starchip from './starship.component';

it('should render Starchip component', () => {
  expect(shallow(<Starchip />)).toMatchSnapshot();
});
