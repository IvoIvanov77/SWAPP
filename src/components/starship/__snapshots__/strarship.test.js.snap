// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`should render Starchip component 1`] = `ShallowWrapper {}`;
